app/src/css/modules/_table.scss

```diff
@@ -34,10 +34,10 @@
         padding: 1em 2em;
         vertical-align: middle;
         overflow: hidden;
-        text-overflow: ellipsis;
-        /*&:first-of-type {
-          padding: 1em 0 1em 2em;
-        }*/
+        min-height: 65px;
```

Comment:
We might not need to add a min-height here if we don't want to. I have it because without it the row will expand vertically when hovered to fit the scrollbar. It does increase all of the row heights though. If we remove it the row will expand on hover, which also might not be terrible. I could go either way.

Reply:
@npauzenga I'm curious what the second option would look like with expand `on hover`.

Reply:
@jjmccoy yeah, if you comment out this line (37) and hover over the table cells, they will expand to fit the scrollbar. Happy to jump on a screenshare if that's helpful.